ngmp/app.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var SwaggerExpress = require('swagger-express-mw');
+var app = require('express')();
+module.exports = app; // for testing
+
+var config = {
+  appRoot: __dirname // required config
+};
+
+SwaggerExpress.create(config, function(err, swaggerExpress) {
+  if (err) { throw err; }
+
+  // install middleware
+  swaggerExpress.register(app);
+
+  var port = process.env.PORT || 10010;
+  app.listen(port);
+
+  if (swaggerExpress.runner.swagger.paths['/hello']) {
```

Comment:
What is it for?

Reply:
This middleware is required for options requests due to CORS . In these requests authentication should not been provided. That's why I added it  https://gist.github.com/cuppster/2344435